public/lesswrong-editor/yarn_something_else.lock

```diff
@@ -278,6 +278,14 @@
     "@ckeditor/ckeditor5-ui" "^31.0.0"
     ckeditor5 "^31.0.0"
 
+"@ckeditor/ckeditor5-mention@31.0.0":
```

Comment:
I think we renamed this a a part of learning more about npm deps, and need to either rename this back or delete the file alltogether